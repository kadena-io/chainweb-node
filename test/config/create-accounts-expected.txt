[TxLog {_txDomain = "USER_test1_accounts", _txKey = "Acct1", _txValue = Object (fromList [("amount",Object (fromList [("_P_decm",Number 1.0e7),("_P_decp",Number 1.0)])),("data",String "Created account"),("balance",Object (fromList [("_P_decm",Number 1.0e7),("_P_decp",Number 1.0)]))])}]