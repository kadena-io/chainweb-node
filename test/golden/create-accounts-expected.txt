- output: HLIEBEUgf32fElyvoriTOISAKAdnV0EGOp6xnyHRb38
  cmd: coinbase
- output: Jjhmj9UsP3_0YBh9A5sOEjAAXH1GBmF7tgnhorcVG_w
  cmd:
    tag: Code
    contents: (test1.create-global-accounts)
